content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,148 @@
+To make changes to a user's rights, first activate Odoo's :ref:`developer mode <developer-mode>`,
+then go to :menuselection:`Settings --> Users & Companies --> Users`. Click on the user to edit.
 
-.. image:: access_rights/groups-inherited.png
+.. image:: access_rights/navigate-to-users-menu.png
    :align: center
-   :height: 330
-   :alt: View of a group’s form emphasizing the tab inherited in Odoo
+   :alt: In developer mode, go to the settings page and click Users and Companies, then Users.
 
-.. important::
-   Remember to always test the settings being changed in order to ensure that they are being applied
-   to the needed and right users.
+At the user's profile page, click on the :guilabel:`Access Rights` tab and scroll down to view the
+current permissions. For each app, use the drop-down menu to select what level of permission this
+user should have. The options vary for each section, yet the most common are: *Blank/None*, *User:
+Own Documents*, *User: All Documents*, or *Administrator*.
 
-The *Menus* tab is where you define which menus (models) the user can have access to.
+Administration includes either :guilabel:`Settings` or :guilabel:`Access Rights`.
 
-.. image:: access_rights/groups-menus.png
+.. image:: access_rights/user-permissions-dropdown-menu.png
    :align: center
-   :height: 330
-   :alt: View of a group’s form emphasizing the tab menus in Odoo
+   :alt: Open the drop-down menu next to each app to set the user's level of permissions.
 
-*Access Rights* rules are the first level of rights. The field is composed of the object name, which
-is the technical name given to a model. For each model, enable the following options as appropriate:
+Creating and modifying groups
+=============================
 
-- *Read*: the values of that object can be only seen by the user.
-- *Write*: the values of that object can be edited by the user.
-- *Create*: values for that object can be created by the user.
-- *Delete*: the values of that object can be deleted by the user.
+*Groups* are app-specific sets of permissions that are used to manage common access rights for a
+large amount of users. Administrators can modify the existing groups in Odoo or create new ones to
+define rules for models within an application.
 
-.. image:: access_rights/groups-access-rights.png
+To access groups, first activate Odoo's :ref:`developer mode <developer-mode>`, then go to
+:menuselection:`Settings --> Users & Companies --> Groups`.
+
+.. image:: access_rights/click-users-and-companies.png
    :align: center
-   :alt: View of a group’s form emphasizing the tab access rights in Odoo
+   :alt: In developer mode, go to the settings page and click Users and Companies, then Groups.
+
+To create a new group, click :guilabel:`New`, select an application, and complete the New Group form
+(detailed below). To modify existing groups, click on a group and edit the contents of the form.
```

Comment:
```suggestion
To create a new group from the :guilabel:`Groups` page, click :guilabel:`Create`. Then, from the
blank group form, select an :guilabel:`Application`, and complete the group form (detailed below).

To modify existing groups, click on an existing group from the list displayed on the
:guilabel:`Groups` page, and edit the contents of the form.
```

Reply:
might be beneficial to also mention/explain what the Name and Share Group fields represent on the Group form (above the tabs)

Reply:
👀